### DISCUSSION TITLE
Dynamic local data source

### DISCUSSION THREAD

**Discussion by bryan-gilbert** [digitalfortress-tech/typeahead-standalone#42] [Ideas]
This is a really nice and clean module that is up to date and doesn't have any dependencies. It is also easy to understand how to set up and use.

In my application provides simulations to the users. Each of these simulations has a set of characters that can be modified during the simulation.  The `local` data source is consumed by the typeahead when the typeahead is initialized and it is not referenced again. The `prefetch` and `remote` data sources fetch data dynamically.  I needed something in between and have submitted a PR for your consideration.

https://github.com/digitalfortress-tech/typeahead-standalone/pull/40

**Comment by niketpathak** [comment]
Hello @bryan-gilbert , thank you for your comment. In case your PR does not pan out, I will try to implement this feature in v5. Thanks again!
